File: TeamYeyDayoTravelAccess/app/src/main/java/com/example/dayotravelaccess/MainActivity.java
package com.example.dayotravelaccess;

import android.content.Intent;
import android.os.Bundle;
import android.view.MenuItem;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import com.example.dayotravelaccess.dashboard;
import com.example.dayotravelaccess.history;
import com.example.dayotravelaccess.transaction;
import com.google.android.material.bottomnavigation.BottomNavigationView;

public class MainActivity extends AppCompatActivity implements BottomNavigationView.OnNavigationItemSelectedListener {

    private BottomNavigationView bottomNavigationView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        bottomNavigationView = findViewById(R.id.bottom_navigation);
        bottomNavigationView.setOnNavigationItemSelectedListener(this);
        bottomNavigationView.setSelectedItemId(R.id.ic_transaction);
    }

    @Override
    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
        Intent intent;
        switch (item.getItemId()) {
            case 1:
                intent = new Intent(MainActivity.this, dashboard.class);
                startActivity(intent);
                return true;
            case 2:
                intent = new Intent(MainActivity.this, transaction.class);
                startActivity(intent);
                return true;
            case 3:
                intent = new Intent(MainActivity.this, account.class);
                startActivity(intent);
                return true;
        }
        return false;
    }
}

File: TeamYeyDayoTravelAccess/app/src/main/java/com/example/teamyey_dayotravelaccess/MainActivity.kt
package com.example.teamyey_dayotravelaccess

import android.annotation.SuppressLint
import android.media.Image
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.MenuItem
import android.widget.Button
import android.widget.ImageView
import android.widget.TextView
import android.widget.Toast
import androidx.fragment.app.Fragment
import com.example.teamyey_dayotravelaccess.fragments.AccountFragment
import com.example.teamyey_dayotravelaccess.fragments.HomeFragment
import com.example.teamyey_dayotravelaccess.fragments.TransactionFragment
import com.google.android.material.bottomnavigation.BottomNavigationView

class MainActivity : AppCompatActivity() {

    private val homeFragment = HomeFragment()
    private val accountFragment = AccountFragment()
    private val transactionFragment = TransactionFragment()


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        replaceFragment(homeFragment)


        val bottom_navigation = findViewById<BottomNavigationView>(R.id.bottom_navigation)
        bottom_navigation.setOnNavigationItemSelectedListener {
            when(it.itemId){
            R.id.ic_home -> replaceFragment(homeFragment)
            R.id.ic_transaction ->replaceFragment(transactionFragment)
                R.id.ic_person -> replaceFragment(accountFragment)


            }
            true
        }


    }

    private fun replaceFragment(fragment: Fragment) {
        if (fragment !=null) {
            val transaction = supportFragmentManager.beginTransaction()
            transaction.replace(R.id.fragment_container, fragment)
            transaction.commit()

        }
    }
}
